app/yarn.lock

```diff
@@ -1642,6 +1642,18 @@ builtin-status-codes@^3.0.0:
   version "3.0.0"
   resolved "https://registry.yarnpkg.com/builtin-status-codes/-/builtin-status-codes-3.0.0.tgz#85982878e21b98e1c66425e03d0174788f569ee8"
 
+bull@^3.4.1:
```

Comment:
(this will probably need to be recommitted after removing `bull` from `app/package.json`)